runtime/adopted_build.go

```
@@ -0,0 +1,848 @@
+		if element == "" || element == "." || element == ".." || element == ".git" {
+			return fmt.Errorf("refused tree path %q", path)
+		}
+	}
+	return nil
+}
+
+// blobID recomputes Git's object id for content, so a materialized file is
+// checked against the id the tree named rather than assumed correct.
+func blobID(content string) string {
+	sum := sha1.New()
+	fmt.Fprintf(sum, "blob %d\x00", len(content))
```

golangci-lint (2.13.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)